### ISSUE TITLE
Problems clicking game in multiplayer window

### ISSUE THREAD

**Issue by dlhoff** [worldofbalance/client#82]
In the screen capture below i try to click on any of the multiplayer games and the button depresses but no line appears showing me as the host, indicating that i am waiting for others to join.  This was working ok before i did a pull of development yesterday.  

Also, entering the lobby is slow because it counts the Days from 1 to ~20 very slowly. This is new in the last week or so. 

Another issue is that sometimes after many plays the top three buttons (Who's online, play
mini games, status) are not clickable. I am not presently having that problem, but it happened 
several times in the past 2 weeks. I fix by rebooting my computer. 

![c2](https://cloud.githubusercontent.com/assets/17733350/15021766/7a1f849c-11dd-11e6-8832-c2ca93fd08fa.PNG)

**Comment by dlhoff** [comment]
I listed 3 problems above. Two of them seemed to be solved/understood. The one for the screen capture was fixed when we rebooted the server. That also fixed the problem with the Day counting up.

But the 3rd problem remains. The 3 buttons at the top of the screen become not-clickable is still with me. It happened again twice today. The only solution i found is to reboot my laptop. Then it works for 7 to 15 sessions with Unity and then it fails again.
